xray/audit/jas/jasmanager.go

```diff
@@ -0,0 +1,89 @@
+package jas
+
+import (
+	"errors"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-core/v2/xray/utils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"github.com/jfrog/jfrog-client-go/xray/services"
+	xrayUtils "github.com/jfrog/jfrog-client-go/xray/services/utils"
+	"os"
+)
+
+const serverDetailsErrorMessage = "cant get xray server details"
+
+var (
+	analyzerManagerExecuter utils.AnalyzerManagerInterface = &utils.AnalyzerManager{}
+	skippedDirs                                            = []string{"**/*test*/**", "**/*venv*/**", "**/*node_modules*/**", "**/*target*/**"}
+)
+
```

Comment:
Hi
The `GetExtendedScanResults()` function, which is the main purpose of creating the jas manager file, is calling to all other scanners and "collects" all the three jas scans results. If it will be moved to the `utils` directory we'll get a circular dependency: `utils/analyzermanager.GetExtendedScanResults() -> xray/audit/jas -> utils/analyzermanager.Exec()`

Reply:
@eyalbe4

Reply:
Understood. Thanks @orto17!